source/about-mongosync.txt

```diff
@@ -0,0 +1,165 @@
+
+.. include:: /includes/fact-state-transitions.rst
+
+To view the current state of ``mongosync``, use the :ref:`/progress
+<c2c-api-progress>` endpoint.
+
+``mongosync`` Diagram
+-----------------------
+
+The following diagram illustrates the relationship between ``mongosync`` states 
+and API operations:
+
+.. figure:: /images/mongosync.drawio.baked.svg
+   :alt: Diagram of relationship between ``mongosync`` states and API operations
+   :figwidth: 600px
+
+Details
+-------
+
+Connect Source and Destination Clusters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You must create an initial connection to the source and destination 
+clusters before ``mongosync`` can start to sync data. For more information, see 
+:ref:`c2c-connecting`.
+
+When ``mongosync`` connects to the source and destination clusters, the 
+synchronization process enters the :ref:`IDLE <c2c-state-idle>` state.  
+
+Start Sync 
+~~~~~~~~~~
+
+To start the synchronization process, call the :ref:`start <c2c-api-start>` 
+endpoint. 
+
+The ``start`` endpoint begins the ``RUNNING`` state, which is when ``mongosync`` 
+syncs data from the source cluster to the destination cluster. If you 
+make subsequent writes to the source cluster, ``mongosync`` applies the 
+operations to the destination cluster. 
+
+For every source collection that syncs, 
+``mongosync`` creates a corresponding collection on the destination. 
+
+``mongosync`` temporarily alters some of the replicated collection
+characteristics in the destination cluster. It changes these destination cluster
+characteristics back to the source cluster characteristics during the ``COMMIT`` 
+state. For more information, see 
+:ref:`Collection Characteristic Changes <c2c-collection-changes>`.
+
+The ``RUNNING`` state includes two phases:
+
+- :ref:`Collection copy <c2c-collection-copy>`
+
+- :ref:`Change event application <c2c-cea>`
+
+.. _c2c-collection-copy:
+
+Collection Copy
+'''''''''''''''
+
+During the collection copy phase, ``mongosync`` executes the initial data 
+transfer from the source cluster to the destination 
+cluster. ``mongosync`` performs this data transfer by splitting the 
+source collections into partitions and copying the partitions to their 
+corresponding destination collections.
+
+.. _c2c-cea:
+
+Change Event Application
+''''''''''''''''''''''''
+
+The change event application (CEA) phase starts after the collection copy phase
+ends. 
+
+During the CEA phase, ``mongosync`` continuously 
+updates the destination cluster when you make changes to the source cluster. 
+
+To make continuous updates to the destination cluster, ``mongosync`` opens a 
+:ref:`change stream <changeStreams>` on the source cluster and applies the
+:ref:`change events <change-events>` it receives to the destination cluster. 
+
+When the CEA phase begins, ``mongosync`` first applies the change events 
+that are received during the collection copy phase before moving on to the 
+events that are received during the CEA phase. 
```

Comment:
[optional/suggestion]

Brevity
```suggestion
When the CEA phase begins, ``mongosync`` applies the change events 
received during the collection copy phase before processing the 
events received during the CEA phase. 
```

Reply:
Thanks for the suggestion, changing now!